knowledge_runtime/knowledge_runtime/main.py

```
@@ -33,6 +34,16 @@ async def lifespan(app: FastAPI):
         log_level=settings.log_level,
     )
 
+    # Initialize database connection for reference mode
+    if settings.DATABASE_URL:
+        try:
+            init_db()
+            logger.info("Database connection initialized for reference mode")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)